### PR TITLE
Enables FISAPT-D

### PR DESCRIPTION
**Pull request by dsirianni**
## Description
Enables FISAPT-D with a global option.

## Checklist
- [x] [All or relevant fraction of full tests run](http://psicode.org/psi4manual/master/build_planning.html#how-to-run-a-subset-of-tests)

## Status
- [x] Ready for review
- [x] Ready for merge

### CODE REVIEW

**Review by loriab — COMMENTED** [review]
Since D doesn’t need it, drop the fisapt_empirical_dispersion for now, I think. That can be handled later via the energy() arg. Also, add the do_fsapt_disp to the optstash, please.

File: psi4/src/read_options.cc [hunk 1]
```diff
@@ -1001,6 +1001,8 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
 
         /*- Do an F-SAPT analysis? -*/
         options.add_bool("FISAPT_DO_FSAPT", true);
+        /*- Do F-SAPT Dispersion? If not, do -D Dispersion -*/
```
Comment (loriab) [inline psi4/src/read_options.cc]: nix the "If not, do -D Dispersion"

File: psi4/driver/procrouting/sapt/fisapt_proc.py [hunk 2]
```diff
@@ -91,7 +91,26 @@ def fisapt_compute_energy(self):
         self.find()
         core.timer_off("FISAPT:FSAPT:ind")
         core.timer_on("FISAPT:FSAPT:disp")
-        self.fdisp()
+        if core.get_option("FISAPT", "FISAPT_DO_FSAPT_DISP"):
+            core.timer_on("FISAPT:FSAPT:disp")
+            self.fdisp()
+            core.timer_off("FISAPT:FSAPT:disp")
+            self.fdrop()
+        else:
```
Comment (loriab) [inline psi4/driver/procrouting/sapt/fisapt_proc.py]: comment out this block please. let's get only what they need in. if they ran with fisapt_do_fsapt_disp=False, then ran a `mol.run_dftd3(... d3mbj)`, I could see someone getting confused over what empirical dispersion to believe.

File: psi4/driver/procrouting/sapt/fisapt_proc.py [hunk 3]
```diff
@@ -91,9 +92,27 @@ def fisapt_compute_energy(self):
...
+        #    text = []
+        #    text.append("   => {}: Empirical Dispersion <=".format(dashD.fctldash.upper()))
+        #    text.append(" ")
+        #    text.append(dashD.description)
+        #    text.append(dashD.dashlevel_citation.rstrip())
+        #    text.append("\n    Empirical Dispersion Energy [Eh] =     {:24.16f}\n".format(Edisp))
+        #    text.append('\n')
+        #    core.print_out('\n'.join(text))
         core.timer_off("FISAPT:FSAPT:disp")
-        self.fdrop()
```
Comment (loriab) [inline psi4/driver/procrouting/sapt/fisapt_proc.py]: does `core.timer_off("FISAPT:FSAPT:disp")` need to go away?

and is `fdrop` solely assoc. with disp?

Comment (dsirianni) [reply psi4/driver/procrouting/sapt/fisapt_proc.py]: Timer needs banished, and `fdrop` not local to dispersion.  Fixed now.

**Review by loriab — APPROVED** [review]
thanks for all the changes!

**Pull request merged**